static/src/javascripts/projects/commercial/modules/sticky-mpu.js

```diff
@@ -36,19 +36,35 @@ const stickyMpu = (adSlot: HTMLElement) => {
     }
 
     fastdom
-        .read(
-            () =>
-                referenceElement[
-                    config.page.hasShowcaseMainElement
-                        ? 'offsetHeight'
-                        : 'offsetTop'
-                ] + adSlot.offsetHeight
+        .read(() => {
+            if (config.page.hasShowcaseMainElement) {
+                return referenceElement.offsetHeight + adSlot.offsetHeight;
```

Comment:
I _think_ this might need to be `referenceElement.offsetTop + referenceElement.offsetHeight + adSlot.offsetHeight` for garnett, not 100%. because the container we are adding the MPU to actually starts before the `referenceElement`, so the top will always count. see the paper diagram on your desk.